hadoop-tools/hadoop-streaming/src/main/java/org/apache/hadoop/streaming/mapreduce/StreamInputFormat.java

```diff
@@ -62,14 +58,8 @@ public RecordReader<Text, Text> createRecordReader(InputSplit genericSplit,
     context.progress();
 
     // Open the file and seek to the start of the split
-    Path path = split.getPath();
-    FileSystem fs = path.getFileSystem(conf);
-    // open the file
-    final FutureDataInputStreamBuilder builder = fs.openFile(path);
-    FutureIOSupport.propagateOptions(builder, conf,
-        MRJobConfig.INPUT_FILE_OPTION_PREFIX,
-        MRJobConfig.INPUT_FILE_MANDATORY_PREFIX);
-    FSDataInputStream in = FutureIOSupport.awaitFuture(builder.build());
+    FileSystem fs = split.getPath().getFileSystem(conf);
+    FSDataInputStream in = fs.open(split.getPath());
```

Comment:
good  q. not sure why i reverted (accidentally?). reverted back.

FWIW, hadoop-streaming is a module i don't think gets used any more, not now we have things like flnk. cutting it would be good